Regression/WarpX-tests.ini

```diff
@@ -53,10 +52,7 @@ branch = 88b868099018c191e7d3ea424d3e3420b826dd69
 [source]
 dir = /home/regtester/AMReX_RegTesting/warpx
 branch = development
-
-[extra-PICSAR]
-dir = /home/regtester/AMReX_RegTesting/picsar/
-branch = 8ff12fbf118b9aba7cfe602cb1a5e6da32bf7eef
+cmakeSetupOpts = -DAMReX_ASSERTIONS=ON
```

Comment:
In GNUmake, we also set `TEST=TRUE`.
This implies:
- signaling NaNs initialized for all MultiFabs (just as in Debug builds)
- ... more?